modules/gdscript/gdscript_parser.cpp

```diff
@@ -1117,7 +1117,6 @@ GDScriptParser::EnumNode *GDScriptParser::parse_enum() {
 				}
 				item.custom_value = value;
 			}
-			item.rightmost_column = previous.rightmost_column;
```

Comment:
This looks like a false positive no?

Reply:
https://github.com/godotengine/godot/blob/2d446771d6701bb77432b6df5a768e53c9a6c95d/modules/gdscript/gdscript_parser.cpp#L1094-L1121

The line above(line 1120) is same as line 1095.
And,it in between them, item.rightmost_column isn't changed.previous.rightmost_column isn't modified either.

Reply:
Ah my bad, it looks like github's formatting tricked me.